src/Pages/DistributedUptime/Details/Components/Chatbot/index.jsx

```
@@ -16,6 +17,7 @@ const MESSAGES = [
 
 const ChatBot = ({ sx }) => {
```

ESLint

[error] 'sx' is missing in props validation

(react/prop-types)